Symfony/CS/Console/Output/ProcessOutput.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Console\Output;
+
+use Symfony\Component\Console\Output\StreamOutput;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\CS\FixerFileProcessedEvent;
+
+class ProcessOutput
+{
+    /**
+     * File statuses map.
+     *
+     * @var array
+     */
+    private static $eventStatusMap = array(
+        FixerFileProcessedEvent::STATUS_UNKNOWN => array('symbol' => '?', 'format' => '%s', 'description' => 'unknown'),
+        FixerFileProcessedEvent::STATUS_INVALID  => array('symbol' => 'I', 'format' => '<bg=red>%s</bg=red>',  'description' => 'invalid file syntax, file ignored'),
+        FixerFileProcessedEvent::STATUS_SKIPPED => array('symbol' => '', 'format' => '<bg=cyan>%s<bg=cyan>', 'description' => ''),
+        FixerFileProcessedEvent::STATUS_NO_CHANGES => array('symbol' => '.', 'format' => '%s', 'description' => 'no changes'),
+        FixerFileProcessedEvent::STATUS_FIXED => array('symbol' => 'F', 'format' => '<bg=green>%s</bg=green>', 'description' => 'fixed'),
+        FixerFileProcessedEvent::STATUS_EXCEPTION => array('symbol' => 'E', 'format' => '<bg=red>%s</bg=red>', 'description' => 'error'),
+        FixerFileProcessedEvent::STATUS_LINT => array('symbol' => 'E', 'format' => '<bg=red>%s</bg=red>', 'description' => 'error'),
+    );
+
+    /**
+     * Stream output instance.
+     *
+     * @var output
+     */
+    private $output;
+
+    /**
+     * Event dispatcher instance.
+     *
+     * @var EventDispatcher
```

Comment:
> Ah ok. I thought it would be good practice to drop references when you no longer need those. 

No, it's not. You don't need to repeat what php's already doing for you under the hood.